README.md

```diff
@@ -50,20 +68,22 @@ In order to run this project you need:
 ### Setup
 Clone this repository to your computer
 
-### Usage
-To run the project, execute the "index.html" file in your browser
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Install
+Install this project with:
 
```

Comment:
- [ ] As specified by the previous reviewer you need to customize the readme template based on the needs of your project. Kindly add instructions on how users can install all dependencies of your project after cloning it into their local machine. Example: 
```
npm install
```